src/exceptions.md

```diff
@@ -214,9 +218,23 @@ Contents of section .vector_table:
  0030 00000000 00000000 79000000 79000000  ........y...y...
 ```
 
+The vector table now resembles the results of all the code snippets in this book so far. To summarize:
+- In the [_Inspecting it_] section of the earlier memory chapter, we learned that:
+    - The first entry in the vector table contains the initial value of the stack pointer.
+    - Objdump prints in `little endian` format, so the stack starts at `0x2001_0000`.
+    - The second entry points to address `0x0000_0041`, the Reset handler.
+        - The address of the Reset handler can be seen in the disassembly above, being `0x40`.
+        - The first bit being set to 1 does not alter the address due to alignment requirements. Instead, it causes the function to be executed in _thumb mode_.
+- Afterwards, a pattern of addresses alternating between `0x79` and `0x00` is visible.
+    - Looking at the disassembly above, it is clear that `0x79` refers to the `DefaultExceptionHandler` (`0x78` executed in thumb mode).
```

Comment:
Could you add another bullet that mentions that all the reserved slots have the zero value in the disassembly?